demo/Titanic/domain_lib/JourneyLib.py

```diff
@@ -1,30 +1,52 @@
 #!/usr/bin/env python3
 
-from simulation.floating_object import FloatingObject
 from system.titanic import Titanic
 from simulation.iceberg import Iceberg
 from simulation.titanic_in_ocean import TitanicInOcean
 from simulation.journey import Journey
 from simulation.ocean import Ocean
 from robot.api.deco import keyword
 
-class JourneyLib():
+
+class JourneyLib:
     def __init__(self):
         self.ocean = Ocean()
+        self.journey = Journey(self.ocean)
 
-    @keyword("Spawn titanic at coordinate N${n} W${w} with a heading of ${heading} and at speed of ${speed} knots")
-    def spawn(self, n:int , w:int, heading:int, speed:int):
-        t = Titanic(speed, "east")
-        tio = TitanicInOcean(t, n, w, speed, "west")
+    @keyword("Spawn titanic at coordinate long ${n} lat ${w} heading ${heading} at speed of ${speed} knots")
```

Comment:
I would prefer to use longitude and latitude instead of long and lat

Reply:
Will update that